docs/platforms/react-native/user-feedback/configuration/index.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: Configuration
+sidebar_order: 6100
+description: "Learn about the User Feedback Widget configuration options."
+---
+
+## User Feedback Widget
+
+The User Feedback Widget offers many customization options, and if the available options are insufficient, you can [use your own UI](/platforms/react-native/user-feedback/#user-feedback-api).
+
+![An image showing the main customization options for the User Feedback Widget](./img/mobile-user-feedback-widget-customization.png)
```

Comment:
This is a cropped version of the [JS image](https://docs.sentry.io/platforms/javascript/user-feedback/configuration/#user-feedback-widget) since the UI and props matched